.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "nightly"
+# command to install dependencies
+install: "pip install -r requirements.txt"
+# command to run tests
+script: nosetests
```

Comment:
I suspect you haven't actually touched this file.  Unless you are using `nose`, in which case I apologize.